doc/book/sql-ddl.md

```diff
@@ -146,17 +146,21 @@ In alphabetical order:
 Type             | Arguments For Construction
 -----------------|---------------------------
 BigInteger       | `$name`, `$nullable = false`, `$default = null`, `array $options = array()`
+Binary           | `$name`, `$length`, `nullable = false`, `$default = null`, `array $options = array()`
```

Comment:
Please undo this changes, because it doesn't belong to the topic of this PR. Create a new PR. Thanks!